src/ngraph/op/op_tbl.hpp

```diff
@@ -106,6 +106,7 @@ NGRAPH_OP(Max, ngraph::op)
 NGRAPH_OP(Maximum, ngraph::op)
 NGRAPH_OP(MaxPool, ngraph::op)
 NGRAPH_OP(MaxPoolBackprop, ngraph::op)
+NGRAPH_OP(Merge, ngraph::op)
```

Comment:
Gerrit checks for INTELGPU backend are switched off. To be able to build INTELGPU backend w/o switch-enum error you need to add "case OP_TYPEID::Merge:" line into switch from src/ngraph/runtime/intelgpu/intelgpu_backend.cpp (between 2050 - 2106 lines)